Cargo.toml

```diff
@@ -40,11 +40,12 @@ version = "2.3.0"
 default-features = false
 
 [features]
-default = ["groups", "pairings", "alloc", "std" ,"endo"]
+default = ["groups", "pairings", "alloc", "std", "endo", "serde_req"]
```

Comment:
I'll not add it by default TBH. Is something that most of the times will not be required/used. I would keep it in a sepparated feature.

Reply:
Agreed, changing that now